code/mob/living/carbon/human/procs/emote.dm

```diff
@@ -41,7 +41,15 @@
 			// most commonly used emotes first for minor performance improvements
 			if ("scream")
 				if (src.emote_check(voluntary, 50))
-					if (!muzzled)
+					if(src.bioHolder && src.bioHolder.HasEffect("mute"))
```

Comment:
```suggestion
					if(src.bioHolder?.HasEffect("mute"))
```

Reply:
oooh i didnt know you could format things like that!